nginx.conf.template

```diff
@@ -0,0 +1,74 @@
+http {
+    include       mime.types;
+    default_type  application/octet-stream;
+
+    sendfile        on;
+
+    keepalive_timeout  60;
+
+    server {
+    	# if no Host match, close the connection to prevent host spoofing
+    	listen 80 default_server;
+    	return 444;
+    }
+
+    server {
+        listen		80;
+        listen		443 ssl;
+
+        # If using ngrok, s
+        ee configuration instructions
+        server_name <your_server_name>;
+        add_header	'Access-Control-Allow-Origin' '*';
+        add_header	'Access-Control-Allow-Methods' 'GET, POST, OPTIONS';
+        add_header	'Access-Control-Allow-Headers' 'Content-Type,Accept';
+
+	    access_log  /usr/local/var/log/nginx/access.log;
+
+        # You need to run 
+        # $ openssl req -newkey rsa:2048 -sha256 -nodes -keyout private.key -x509 -days 3650 -out cert.pem
+        ssl_certificate     </path/to/cert/file>;
+        ssl_certificate_key </path/to/key/file>;
+
+        location / {
+            # If using ngrok, it requires that a web application is running on localhost 
+            # and listening on the ngrok port
+            proxy_pass https://localhost:<ngrok-port>;
+            proxy_redirect    off;
+            proxy_http_version  1.1;
+            proxy_set_header    Upgrade $http_upgrade;
+            proxy_set_header    Connection keep-alive;
+            proxy_set_header    Host $host;
+            proxy_cache_bypass  $http_upgrade;
+            proxy_set_header    X-Forwarded-For $proxy_add_x_forwarded_for;
+            # enable this if and only if you use HTTPS
+            proxy_set_header    X-Forwarded-Proto https;
+            proxy_set_header    X-Real-IP $remote_addr;    
+            proxy_set_header    X-Scheme $scheme;
+            proxy_connect_timeout       600;
+            proxy_send_timeout          600;
+            proxy_read_timeout          600;
+            send_timeout                600;
+        }
+
+        location /<your-bot-token> {
+            proxy_pass https://localhost:<ngrok-port>;
+            proxy_redirect    off;
+            proxy_http_version  1.1;
+            proxy_set_header    Upgrade $http_upgrade;
+            proxy_set_header    Connection keep-alive;
+            proxy_set_header    Host $host;
+            proxy_cache_bypass  $http_upgrade;
+            proxy_set_header    X-Forwarded-For $proxy_add_x_forwarded_for;
+            proxy_set_header    X-Forwarded-Proto https;
+            proxy_set_header    X-Real-IP $remote_addr;    
+            proxy_set_header    X-Scheme $scheme;
+            proxy_connect_timeout       600;
+            proxy_send_timeout          600;
+            proxy_read_timeout          600;
+            send_timeout                600;
+        }
+    }
+
+    include servers/*;
+}
```

Comment:
Is this template necessary?

Reply:
Thought it necessary in case someone else wants to work with the bot.